### PR TITLE
relax dependency on synced gem, not used directly by this gem

### PR DESCRIPTION
**Pull request by Mandaryn**

### CODE REVIEW

File: spec/controllers/admin/base_controller_spec.rb [hunk 1]
```diff
@@ -9,9 +9,6 @@ def index
 
   let(:account) { create :account }
 
-  before do
-    allow(controller).to receive(:setup_request)
```
Comment (Azdaroth) [inline spec/controllers/admin/base_controller_spec.rb:13]: I guess this could go to another PR or commit at least, not directly related to `synced` I think.

**Comment by Fredar** [comment]
Looks ok

File: spec/controllers/admin/base_controller_spec.rb (hunk 1 above)
Comment (Mandaryn) [inline spec/controllers/admin/base_controller_spec.rb:13]: this broke down with some gem update, rails maybe?, but seems not necessary anyway
